packages/api/migrations/migration_4.js

```diff
@@ -0,0 +1,38 @@
+const { Manager, Rule } = require('../models');
```

Comment:
This migration adds a logEventArn via the (kinesis) rule model code for any rule missing it's logEventArn.